frontend/ui/elements/screen_rotation_menu_table.lua

```diff
@@ -60,6 +60,44 @@ When unchecked, the default rotation of the file browser and the default/saved r
             separator = true,
         })
 
+        local function image_rotation_item(setting, label, separator)
+            return {
+                text = label,
+                radio = true,
+                separator = separator or false,
+                checked_func = function()
+                    return (G_reader_settings:readSetting("image_clockwise_rotation", true) == setting)
+                end,
+                callback = function(touchmenu_instance)
+                    G_reader_settings:saveSetting("image_clockwise_rotation", setting)
+                    G_reader_settings:flush()
+                    if touchmenu_instance then touchmenu_instance:updateItems() end
+                end,
+
+            }
+        end
+        table.insert(rotation_table, {
+            text = _("Image viewer rotation"),
+            help_text = _("Settings concerning the rotation of images in the full-screen image viewer"),
+            separator = true,
+            sub_item_table = {
+                image_rotation_item(true, "Clockwise"),
+                image_rotation_item(false, "Counter-clockwise", true),
+                {
+                    text = "Auto rotate for best fit",
+                    help_text = _("Rotate the image to better fit the screen if the aspect ratios of the screen and the image are misaligned (landscape screen and portrait image or vice-versa)."),
```

Comment:
I don't think "misaligned" is the right word.
`Auto-rotate image in image viewer to best match aspect ratios (ie. rotate a landscape image when the device is in portrait mode).` should be enough.